### ISSUE TITLE
How to implement the Multi-resolution net structure

### ISSUE THREAD

**Issue by buweiaini** [anson0910/CNN_face_detection#32]
Dear anson0910 :  
         Could you please show me an example of how to implement the Multi-resolution net structure?
I really don't know how to modify the .ptototxt file to implement,  thank you very much!!

**Comment by anson0910** [comment]
I believe this can be done by the following steps:
1. Add the 12-net layers to the 24-net .prototxt files (concatenating the fully connected layers as shown in the paper)
2. Transplant the trained 12-net parameters to the multi-resolution net, and set the local learning rate to 0, as shown [here](https://groups.google.com/forum/#!msg/caffe-users/wfwl468OpG4/JlnnCcaEboYJ)
3. Train the net

**Comment by buweiaini** [comment]
Dear  anson0910 :
        Thank you for your reply!  
        I am sorry to trouble you again . When I train the caffemodel of 12-net,  I used 18,000 positive faces collected from the AFLW datast, and about 180,000 non_face patchs, the model converge ,,,,,but when I use the 12-net caffemodel to test , the confidence of detect windows are all 0.5,,,,,,,,,then I augument the number of positives from18,000 to 54,000,,,,I get the same result,,,,,,,
       Have you ever met this kind of problem?
      Thank you !

**Comment by anson0910** [comment]
If during training, the accuracy on the validation set is higher than 0.9, 12-net should perform great during testing. 
However, if the accuracy stays fixed at 0.5, you may need to stop the training, and start over again.
It should be an issue related to the initialized parameters' range or learning rate, but just retraining a few times should give the desired result!

**Comment by buweiaini** [comment]
dear anson0910:
     if i want to implement the multi-resolution net  ,  how can i train the net?  
i mean we must put a pair of images to the net, so how can we confirm that the each image in the pair can be pass into the net at the same time ?

**Comment by anson0910** [comment]
Hi, 
I think you can simply create the lmdb files as before, and for example in face_24's [train_val](https://github.com/anson0910/CNN_face_detection_models/blob/master/face_24c/train_val.prototxt) file, add a data layer specifying the **data_param**.
